### PR TITLE
Fixing some mistakes on the new post processing jobs + time stamp on cache

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: grakn-engine/src/main/java/ai/grakn/engine/cache/EngineCacheDistributed.java [hunk 1]
```diff
@@ -220,6 +221,20 @@ public void clearAllJobs(String keyspace) {
...
         clearAllJobs(CASTING_JOB, keyspace);
     }
+
+    @Override
+    public long getLastTimeJobAdded() {
+        Optional<Object> value = getObjectFromZookeeper(ENGINE_CACHE_UPDATE_TIME);
+        if(value.isPresent()){
+            return (long) value.get();
+        }
+        return 0L;
```
Comment (aelred) [inline grakn-engine/src/main/java/ai/grakn/engine/cache/EngineCacheDistributed.java]: Maybe:
`return getObjectFromZookeeper(ENGINE_CACHE_UPDATE_TIME).orElse(0L);`
but you might need a nasty cast...

File: grakn-graph/src/main/java/ai/grakn/graph/internal/TypeImpl.java [hunk 2]
```diff
@@ -150,7 +150,9 @@ V addInstance(Schema.BaseType instanceBaseType, BiFunction<Vertex, T, V> produce
         }
 
         Vertex instanceVertex = getGraknGraph().addVertex(instanceBaseType);
-        getGraknGraph().getConceptLog().addedInstance(getLabel());
+        if(!Schema.MetaSchema.isMetaLabel(getLabel())) {
+            getGraknGraph().getConceptLog().addedInstance(getLabel());
+        }
```
Comment (aelred) [inline grakn-graph/src/main/java/ai/grakn/graph/internal/TypeImpl.java]: When are we making instances of meta-types?

Comment (fppt) [inline grakn-graph/src/main/java/ai/grakn/graph/internal/TypeImpl.java]: `graknGraph.admin().getMetaRuleInference().putRule(lhs, rhs);` :)

File: grakn-engine/src/main/java/ai/grakn/engine/cache/EngineCacheDistributed.java (hunk 1 above)
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/cache/EngineCacheDistributed.java]: better yes

**Pull request merged**